### ISSUE TITLE
Lighting Modded forge 1.18.2 + Shaders

### ISSUE THREAD

**Issue by CamiloHerrera13** [Asek3/Oculus#118]
### What happened?

Oculus + Rubidium won't render lights  correctly at dark levels, even when the player is located at a completely dark location like a dark rectangle with 0 lights . When testing optifine, the shaders worked correctly with every setting exactly the same. The issue only happens when shaders are involved, Oculus + Rubidium renders fine when isolated.  No shaders render correctly at dark light levels, attempted with ComplementaryShaders_v4.5 / ComplementaryReimagined_r1.1.1 / ildur's+Vibrant+Shaders+v1.32+Medium .

### Screenshots

![2022-07-02_10 51 59](https://user-images.githubusercontent.com/54414502/177009601-1cd8361a-1184-49db-b775-4570a4a8b7b0.png)
![2022-07-02_10 52 04](https://user-images.githubusercontent.com/54414502/177009603-01c354c4-454f-47f1-8395-4af3521de11c.png)
![2022-07-02_10 35 50](https://user-images.githubusercontent.com/54414502/177009604-907ee45a-dd05-4c6a-9ad3-cce05a5a3304.png)
![2022-07-02_10 36 12](https://user-images.githubusercontent.com/54414502/177009606-2a0617f4-b233-45e1-a1a1-8c9cfccb8269.png)
![Uploading 2022-07-02_11.17.19.png…]()

### Minecraft Version

Minecraft 1.18.2

### Iris Version

oculus-mc1.18.2-1.2.5

### Sodium Version

rubidium-0.5.2a

### Operating System

Windows 10

### What is your GPU?

Nvidia Geforce 980

### Java Version

Java 16

**Comment by CamiloHerrera13** [comment]
[2022-07-02-2.log.gz](https://github.com/Asek3/Oculus/files/9033381/2022-07-02-2.log.gz)

**Comment by PenumbraStar** [comment]
Just want to clarify before I upload my log, is the issue dark areas appearing fully lit? I have been having a similar issue, but afterwards oculus kills the shader rendering pipeline all together.

**Comment by CamiloHerrera13** [comment]
Yes, I have the same issues but tha game works completely normal outside of the lighting issues when dark

**Comment by PenumbraStar** [comment]
So I found the issue was something with PBR and Oculus, not sure what exactly yet...

**Comment by CryoByte33** [comment]
Just wanted to pitch in to say that I have this exact issue. Shaders off, the world appears dark as it should. Shaders enabled and everything is full-bright at all times, despite trying every setting.

Note: I DO have Create installed, but the flywheel backend is disabled, and the same issue persists with Create disabled completely.

Edit: I found the mod giving me an issue. It was "No Night Vision Flashing" by TeamCoFH. Removing it normalized the lighting.

**Comment by DoctorMage** [comment]
Can confirm it also happens in 1.16.5 with the following mods
rubidium
no nv flash
oculus

**Comment by LunarDev-git** [comment]
This is still happening to me on 1.19.2
